TimeEvaluate.go

```diff
@@ -0,0 +1,42 @@
+package cmds
+
+import (
+	"fmt"
+	"time"
+)
+
+type TimeEvaluate struct {
```

Comment:
How to use this inside Run() function

Reply:
in Run() function, just call res.RecordEvent("GetConfig") , it will record the time and can be shown if the timeout error occur.